README.md

```diff
@@ -88,14 +88,26 @@ your Fluo repo or if you just want just want a fresh install.  To view your inst
 cd install/fluo-1.0.0-beta-1-SNAPSHOT
 ```
 
-From here you can run the `fluo` command to administer Fluo.
+Verify your installation by running the `fluo` command which you can use to administer Fluo.
 
 ```
 bin/fluo
 ```
 
-Next, follow the instructions starting at the [Configure a Fluo application][2] section of the 
-Fluo production setup instructions to configure, initialize, and start a Fluo application
+Running Fluo applications
+-------------------------
+
+There are two ways to run Fluo applications using `fluo-dev`:
+
+1. Run pre-built Fluo applications like [fluo-stress] that are listed and configured 
+   in `conf/application.props` and can be run by a single command.
+
+    ```
+    fluo-dev run stress
+    ```
```

Comment:
Could mention where app will run from, for example stress ran from`install/fluo-app-repos/stress`.